tests/phpunit/tests/block-template.php

```diff
@@ -431,6 +431,33 @@ public function test_get_block_templates_paths_dir_doesnt_exists() {
 		$this->assertSame( array(), $template_paths );
 	}
 
+	public function test_get_block_templates_from_registry() {
```

Comment:
Docblock:

:1234: This applies to each of the new tests.

* `@ticket` annotation
* `@covers` annotations
* Description of the test

Reply:
Thanks for the feedback, I added docblocks to all tests in f6a3dfa689ecb1afb8c2cef61a69a3dad6fedbe9.